gotfparse/pkg/converter/converter.go

```diff
@@ -181,3 +271,85 @@ func (t *terraformConverter) SetDebug() {
 func (t *terraformConverter) SetStopOnHCLError() {
 	t.parserOptions = append(t.parserOptions, parser.OptionStopOnHCLError(true))
 }
+
+func (t *terraformConverter) visitModule(m *terraform.Module, out *gabs.Container) {
+	path := t.getModulePath(m)
+
+	for _, b := range m.GetBlocks() {
+		t.visitBlock(b, path, out)
+	}
+}
+
+func getModName(b *terraform.Block) string {
+	// This field is unexported, but necessary to generate the path of the
+	// module. Hopefully aquasecurity/defsec exports this in a future release.
```

Comment:
Please make an upstream issue and/or link to an already existing upstream issue and add that as the note on the comment.

Reply:
Can do. I think we need to upgrade our library before I submit that; we're about 2.5 months and like ... 30 releases out of date. It's important because I think there are a few other issues unexported variables (and a few that used to be exported that aren't now), but I can do that as a follow up to this PR.